config/internal/common/tekton/policy.yaml.tmpl

```diff
@@ -64,6 +64,12 @@ spec:
             matchLabels:
               app: ds-pipeline-metadata-writer-{{.Name}}
               component: data-science-pipelines
+        - podSelector:
+            matchLabels:
+              opendatahub.io/dashboard: 'true'
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: {{.Namespace}}
```

Comment:
this required? iirc podSelector will default to the current namespace if namespaceSelector is not set: 

```
podSelector: io.k8s.apimachinery.pkg.apis.meta.v1.LabelSelector
defined in: Kubernetes v1.25.4+77bec7a 
 
This is a label selector which selects Pods. This field follows standard label selector semantics; if present but empty, it selects all pods.
If NamespaceSelector is also set, then the NetworkPolicyPeer as a whole selects the Pods matching PodSelector in the Namespaces selected by NamespaceSelector. Otherwise it selects the Pods matching PodSelector in the policy's own Namespace
```

[[1]](https://docs.openshift.com/container-platform/4.13/rest_api/network_apis/networkpolicy-networking-k8s-io-v1.html#spec-ingress-from-2)

Reply:
The original thought behind adding a namespaceSelector AND podSelector clause was to make sure we allow the right notebook pod (or rather just the notebook pod deployed in the DSP namespace/workbench), as the notebook pod doesn't have an exclusive label and we're selecting it based on a generic dashboard label here.

But yes, if podSelector defaults to the current ns then it should be safe to take this out? wdyt @gregsheremeta ?

Reply:
> podSelector will default to the current namespace if namespaceSelector is not set

ah, cool, we didn't know that. @DharmitD will test :)

Reply:
Done, tested this out locally by taking out the namespaceSelector and curling from a notebook pods in a separate namespace other than one where the DSPA APIServer exists, thus ensuring that the notebook pod from a different namespace doesn't connect to the DSP APIServer,